tests/integration_tests_plugins/dockercompute/dockercompute/operations.py

```diff
@@ -25,7 +26,21 @@ def start(ctx, **_):
     install_agent_script = ctx.agent.init_script({'user': 'cfyuser'})
     with tempfile.NamedTemporaryFile(mode='w', delete=False) as f:
         f.write(install_agent_script)
-    subprocess.check_call(['bash', f.name])
+
+    # daemonize the script, so that we're not a child of the mgmtworker,
+    # so the agent is not killed when the mgmtworker dies
+    if os.fork() == 0:
+        if not os.path.exists('/tmp/supervisor.sock'):
+            # under systemd, also escape the mgmtworker's cgroup
+            subprocess.check_call([
+                'sudo', '/usr/bin/bash', '-c',
```

Comment:
this _can_ well run under supervisord as well, but it's the only `sudo` call in the whole thing, so I'm adding the `if` so that it doesn't require sudo at all under supervisord.
(we are allowing sudo in the allow_agents fixture)